subscription-service/src/main/kotlin/com/egm/stellio/subscription/web/SubscriptionHandler.kt

```diff
@@ -112,17 +115,20 @@ class SubscriptionHandler(
     fun update(req: ServerRequest): Mono<ServerResponse> {
         val subscriptionId = req.pathVariable("subscriptionId")
         return req.bodyToMono<String>()
+            .flatMap { input ->
```

Comment:
You can also review this one and the following in the same way than the one in the `create` handler